src/ansys/geometry/core/plotting/plotter.py

```diff
@@ -389,7 +394,11 @@ def add(
         elif isinstance(object, Sketch):
             self.plot_sketch(object, **plotting_options)
         elif isinstance(object, Body) or isinstance(object, MasterBody):
-            self.add_body(object, merge_bodies, **plotting_options)
+            if filter:
```

Comment:
IMO, the filter should apply to everything, not only bodies but also components, and other objects that may come in the future (i.e. beams)

Reply:
Changed, now the filter is applied to every object that has `name` attribute.